test/fixed_red_black_tree_view_test.cpp

```diff
@@ -0,0 +1,162 @@
+#include "fixed_containers/concepts.hpp"
+#include "fixed_containers/consteval_compare.hpp"
+#include "fixed_containers/fixed_red_black_tree_view.hpp"
+#include "fixed_containers/fixed_set.hpp"
+
+#include "mock_testing_types.hpp"
+#include "test_utilities_common.hpp"
+
+#include <algorithm>
+#include <cmath>
+#include <cstring>
+#include <gtest/gtest.h>
+#include <ranges>
+
+namespace fixed_containers
+{
+
+static_assert(std::forward_iterator<FixedRedBlackTreeRawView::Iterator>);
```

Comment:
For compatibility with ranges, let's also add:
```C++
static_assert(std::ranges::forward_range<Type>);
static_assert(ranges::forward_range<Type>);
```

Reply:
ec20409